conf/config.yaml

```
@@ -363,3 +363,21 @@ mcp:
   similarity_threshold: 0.8
   download_timeout: 30.0
   max_download_bytes: 104857600  # 100 MiB
+  max_download_bytes: 104857600  # 100 MiB
```

YAMLlint (1.37.1)

[error] duplication of key "max_download_bytes" in mapping

(key-duplicates)